packages/airnode-examples/integrations/coingecko-signed-data/README.md

```diff
@@ -43,7 +43,7 @@ Before making the request, you need to replace the following placeholders:
 The correct command may look like this:
 
 ```sh
-curl -X POST -H 'Content-Type: application/json' -d '{"encodedParameters": "0x3173000000000000000000000000000000000000000000000000000000000000636f696e49640000000000000000000000000000000000000000000000000000626974636f696e00000000000000000000000000000000000000000000000000"}' 'https://am6ncplkx4.execute-api.us-east-1.amazonaws.com/v1/0xfb87102cdabadf905321521ba0b3cbf74ad09c5d400ac2eccdbef8d6143e78c4'
```

Comment:
I think the `coingecko-testable` integration needs this too.

Reply:
https://github.com/api3dao/airnode/pull/1535